config/config.defaults.yml

```diff
@@ -193,6 +193,7 @@ default: &default_settings
   # If you need for some reason change the session key used in cookies, you can do it here.
   # Normally no need to touch this.
   session_key: _sharetribe_session
```

Comment:
I would add a deprecation comment here, saying that this variable is only needed to upgrade from 5.6.0 to 5.7.0 and will be removed soon.